execution/src/eth_token.rs

```diff
@@ -169,7 +169,7 @@ pub fn mint_eth<G: GasMeter>(
             module_storage,
         )
         .map_err(|e| {
-            println!("{e:?}");
+            tracing::error!("mint_eth error: {e:?}");
```

Comment:
Line 291 also has a similar error printing.

Reply:
Thanks! Done in https://github.com/UmiNetwork/op-move/pull/446/commits/bfce6a6dd0f429f70674afc2eaafc427e736c89f